README.rst

```diff
@@ -835,7 +835,7 @@ Choose from the list of available rules:
 
   *Risky rule: risky when any of the functions are overridden.*
 
-* **method_argument_space** [@PSR2, @Symfony, @PhpCsFixer]
+* **method_argument_space** [@PSR2, @Symfony, @PhpCsFixer, @PHP73Migration]
```

Comment:
how is it related to 7.3 ?

Reply:
the `after_heredoc` option is desired while migrating to php 7.3. That is the reason, why I've added the fixer with that option to the ruleset.